components/console-backend-service/internal/domain/apigateway/apirule_resolver.go

```diff
@@ -0,0 +1,145 @@
+package apigateway
```

Comment:
Please rename files from 
The same goes to others so:
- from apigateway.go to apirule.go
- from apirule_resolver.go to resolver.go
- from apirule_service.go to service.go
- from apirule_converter.go to converter.go (or move methods to apirule/convert.go)

Reply:
done